src/components/Balance/SyncBalanceButton/SyncBalanceButton.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react";
+import "./SyncBalanceButton.scss";
+import { Box } from "@material-ui/core";
+import CustomButton from "../../CustomButton";
+import { FormattedMessage } from "react-intl";
+import tradeApi from "../../../services/tradeApiClient";
+import useStoreSessionSelector from "../../../hooks/useStoreSessionSelector";
+import { useDispatch } from "react-redux";
+import { showErrorAlert, showSuccessAlert } from "../../../store/actions/ui";
+import SyncIcon from "@material-ui/icons/Sync";
+import { getDailyUserBalance } from "store/actions/user";
+
+/**
+ * @typedef {import('../../../services/tradeApiClient.types').ExchangeConnectionEntity} ExchangeConnectionEntity
+ * @typedef {Object} DefaultProps
+ * @property {ExchangeConnectionEntity} selectedExchange
+ */
+/**
+ * Provides the navigation bar for the dashboard.
```

Comment:
I know these component descriptions are tedious, we can maybe just put "SyncBalanceButton", or a generic description. Maybe it's possible to tweak the lint rules to allow no description.

Reply:
fixed